pycryptobot.py

```diff
@@ -937,15 +903,16 @@ def executeJob(sc, app=PyCryptoBot(), state=AppState(), trading_data=pd.DataFram
                 # if not live
                 else:
                     if app.isVerbose() == 0:
```

Comment:
Don't get why we are only calculating margin when application is not in verbose mode on sim mode?

Reply:
i can move it to be above the verbose check so it calculates regardless, but is this causing any issues?

Reply:
Mario this is my bad, I see that the point here is we are only calculating when we need those values.
And now I think is we need to switch app.isVerbose() == 0: block with else: block because we are giving detailed output on non verbose mode.